driver/Dockerfile

```diff
@@ -6,10 +6,9 @@ FROM marketplace.gcr.io/google/ubuntu1804
 RUN apt-get update && apt-get install -y git
 
 # Install Miniconda.
-RUN curl -o miniconda.sh https://repo.anaconda.com/miniconda/Miniconda3-py37_4.9.2-Linux-x86_64.sh
-RUN bash miniconda.sh -b
-RUN rm miniconda.sh
+RUN curl -o miniconda.sh https://repo.anaconda.com/miniconda/Miniconda3-py37_4.9.2-Linux-x86_64.sh && bash miniconda.sh -b && rm miniconda.sh
```

Comment:
I'd generally still break it into multiple lines for readability:

```
RUN curl -o miniconda.sh https://repo.anaconda.com/miniconda/Miniconda3-py37_4.9.2-Linux-x86_64.sh && \
    bash miniconda.sh -b && \
    rm miniconda.sh
```

Minor thing and feel free to leave it as is; just would be good to be on the same page with regards to Dockerfile style too.